File: 2024/programacao_avancada/2024.1/07-06-2024/pontuacao.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int e, a,c;
    cin >> e >> a >> c;
    int soma = (e*2) + (a*3) + (c*5);
    if (soma >= 200){
        cout << "O" << "\n";
    } else if (soma >= 150){
        cout << "S" << "\n";
    } else if (soma >= 100){
        cout << "B" << "\n";
    } else {
        cout << "N" << "\n";
    }
    return 0;
}

File: 2024/introducao_programacao/2024.1/07-06-2024/pontuacao.py
e = int(input())
a = int(input())
c = int(input())
soma = (e * 2) + (a * 3) + (c * 5)
if soma >= 200:
    print("O")
elif soma >= 150:
    print("S")
elif soma >= 100:
    print("B")
else:
    print("N")